db/migrations/3_claim_ticker_to_id.sql

```diff
@@ -0,0 +1,52 @@
+-- NOTE: This migration should only run after all pre 7.0 blocks have been processed
+-- Similar logic should be triggered when the 7.0 upgrade is reached
+
+-- Update claims ID field to use asset IDs
+WITH spec AS (
+    SELECT spec_version_id as version
+    FROM blocks
+    ORDER BY datetime DESC
+    LIMIT 1
+)
+UPDATE claims
+SET id =
+    REGEXP_REPLACE(id, '/[^/]+$', '/' || (scope->>'assetId')::TEXT),  -- Replace text after the last '/'
+    scope = jsonb_set(scope, '{value}', to_jsonb(scope->>'assetId'))
```

Comment:
`value` and `assetId` were both kept to support dual version. If you update the value to `assetId` we would require changes in SDK side to return ticker value for old claims.